### DISCUSSION TITLE
git-lfs push in chunks

### DISCUSSION THREAD

**Discussion by brandoarguello** [git-lfs/git-lfs#4834] [Q&A] [answered]
Hi

We have an unusual use case to push LFS content > 20 Gigs using an Azure Application Gateway WAF v2 that limits by policy files > 4Gigs. 
Is there a Git LFS configuration that would allow for chunking the push t 4 Gig or less?

I've looked at config options from https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-config.5.ronn 

Thank you.

**Comment by bk2204** [comment, marked as answer]
Hey,

There's a proposal to allow uploads in chunks, but it's presently not implemented.  It may be that you can find an alternate client that implements it, but the one in this repo doesn't, so there's no supported way to do that.  You'd need to adjust the firewall to permit access or use some sort of proxy that the firewall can't inspect.

**Reply by brandoarguello** [reply]
Thank you Brian!
